crash/subsystem/storage/blocksq.py

```diff
@@ -14,7 +13,7 @@ class NoQueueError(RuntimeError):
 
 types = Types(['struct request'])
 
-def for_each_request_in_queue(queue: gdb.Value) -> Iterable[gdb.Value]:
+def sq_for_each_request_in_queue(queue: gdb.Value) -> Iterable[gdb.Value]:
```

Comment:
I don't have a problem with renaming it, but we should retain the alias. Code that wants to use both interfaces can be changed to use the prefix but this breaks existing scripts.  It's possible to use both, even with the old name, in the same script by doing e.g. import crash.subsystem.storage.blocksq as blocksq and then using blocksq.for_each_request_in_queue.

Reply:
I agree this commit breaks it but following commits do add for_each_request_in_queue() again which is then properly distinguishing sq and mq queues and calling appropriate functions. That being said I've now realized the function will come from a different module (crash/subsystem/storage/block.py not blocksq.py) so scripts using this will indeed be broken. But won't it be better to fix scripts - IHMO we should have the external block layer API in block.py and leave blocksq and blockmq for low level stuff of sq / mq queues. What do you think?